### DISCUSSION TITLE
Music metadata

### DISCUSSION THREAD

**Discussion by User670** [26F-Studio/Techmino#316] [Document]
***WARNING: This post has been edited by someone OTHER THAN User670 (author of this post). Not all info was fact-checked by User670, and User670 is not responsible for anything he didn't write. You can view this thread's edit history.***

| Title            | In-game metadata | Remarks                                                      |
| ---------------- | ---------------- | ------------------------------------------------------------ |
| 1980s            | C₂₉H₂₅N₃O₅       | *Korobeiniki* ("Tetris theme") remix<br />:heavy_check_mark: *Korobeiniki* is (should be?) in public domain.<br />:warning: *Korobeiniki* is a The Tetris Company trademark. However, this arrangement uses a version different from the trademark.<br />One historical version used to contain "Music 2" from Tetris (NES), but was later removed. |
| Antispace        | MrZ              | *Blank* remix                                                |
| 8-bit Happiness  | MrZ              |                                                              |
| Battle           | Aether & MrZ     |                                                              |
| Beat5th          | MrZ              |                                                              |
| Blank            | MrZ (old works)  | The original title screen theme                              |
| Cruelty          | MrZ (old works)  |                                                              |
| Distortion       | MrZ              |                                                              |
| Down             | MrZ              |                                                              |
| Dream            | MrZ              |                                                              |
| Echo             | MrZ              |                                                              |
| Empty            | ERM              | *Blank* remix                                                |
| End              | MrZ (old works)  |                                                              |
| Exploration      | MrZ              | *Blank* remix                                                |
| Far              | MrZ              |                                                              |
| Final            | MrZ (old works)  |                                                              |
| Hang Out         | MrZ              |                                                              |
| Here             | MrZ              |                                                              |
| Hope             | MrZ              |                                                              |
| How Feeling      | V.A.             | Remix of [*Nee Ima Donna Kimochi*](https://www.youtube.com/watch?v=tBja9_KM8Q8) by [Suzuki Yuyuuta](https://www.youtube.com/channel/UCNMG8dXjgqxS94dHljP9duQ). <br />:heavy_check_mark: Video description says "無断使用OK", which Google translate says it means "Unauthorized use OK". So his name was hidden because he is controversial. |
| Infinite         | MrZ (old works)  |                                                              |
| Jazz Nihilism    | Trebor           | *Blank* remix                                                |
| Malate           | ZUN & C₂₉H₂₅N₃O₅           | Remix of *Bad apple!!* |
| Magicblock       | MrZ              | *Korobeiniki* ("Tetris theme") remix<br />:heavy_check_mark: *Korobeiniki* is (should be?) in public domain.<br />:warning: *Korobeiniki* is a The Tetris Company trademark. |
| Memory           | MrZ              |                                                              |
| Moonbeam         | Beethoven & MrZ  | remix/adaptation of 3rd movement of Piano Sonata No. 14 in C-sharp minor ("Moonlight Sonata") by Ludwig van Beethoven<br />:heavy_check_mark: Piano Sonata No. 14 in C-sharp minor is in public domain. |
| New Era          | MrZ              |                                                              |
| Nil              | MrZ              | *Blank* remix                                                |
| Null             | MrZ              | *Blank* remix/adaptation                                     |
| Overzero         | MrZ              | *Blank* remix                                                |
| Oxygen           | MrZ              |                                                              |
| Peak             | MrZ              |                                                              |
| Push             | MrZ (old works)  |                                                              |
| Race             | MrZ (old works)  |                                                              |
| Race Remix       | 柒栎流星         | *Race* remix                                                 |
| Reason           | MrZ (old works)  |                                                              |
| Rectification    | MrZ              | *Distortion* remix                                           |
| Rockblock        | MrZ              | *Korobeiniki* ("Tetris theme") remix<br />:heavy_check_mark: *Korobeiniki* is (should be?) in public domain.<br />:warning: *Korobeiniki* is a The Tetris Company trademark. |
| Sakura           | ZUN & C₂₉H₂₅N₃O₅ | Remix of *Bloom Nobly, Ink-Black Cherry Blossom ~ Border of Life* and *Border of Life* by ZUN (*Oota Junya*) (in *Perfect Cherry Blossom*).<br/>:heavy_check_mark:ZUN allows fan work and remix of the Touhou music as long appropriate credits are given. |
| Secret7th        | MrZ              |                                                              |
| Secret7th Remix  | 柒栎流星         | *Secret7th* remix                                            |
| Secret8th        | MrZ              |                                                              |
| Secret8th Remix  | MrZ              | *Secret8th* remix                                            |
| Shift            | MrZ              |                                                              |
| Shining Terminal | MrZ              |                                                              |
| Space            | MrZ              | *Blank* remix                                                |
| Spring Festival  | MrZ              | One historical version include direct quote from *Spring Festival Overture* by Li Huanzhi. Current version only quotes the first four notes. |
| Storm            | MrZ              |                                                              |
| Sugar Fairy      | MrZ              | remix of *Dance of the Sugar Plum Fairy* ("NES Tetris Music A") by Pyotr Ilyich Tchaikovsky<br />:heavy_check_mark: *Dance of the Sugar Plum Fairy* is (should be?) in public domain. |
| Super7th         | MrZ              | *Secret7th* remix                                            |
| There            | MrZ              |                                                              |
| Truth            | MrZ              |                                                              |
| Vacuum           | MrZ              | *Blank* remix                                                |
| Vapor            | MrZ              |                                                              |
| Venus            | MrZ              |                                                              |
| Warped           | MrZ              |                                                              |
| Waterfall        | MrZ              |                                                              |
| Way              | MrZ              |                                                              |
| Xmas             | MrZ              | *We Wish You a Merry Christmas* remix<br />:heavy_check_mark: *We Wish You a Merry Christmas* is (should be?) in public domain. |

**Comment by MrZ626** [comment]
Suzuki Yuyuuta is a controversial figure, so I hide his name with "????" on purpose.
It's not disrespecting.

**Reply by C29H25N3O5** [reply]
How about revealing his name by clicking on the “V.A.” multiple times 🤔

[a comment by an automated account is not shown]

**Reply by User670** [reply]
The thing you described is about **copyright** - by not using their recordings, you are fine in the **copyright** department.

I don't know how **trademark**-related laws deals with Techmino's usage of the song though.

**Comment by MrZ626** [comment]
I have a document listed "music - using modes", do you need this? @user670

**Reply by User670** [reply]
That's not what this post is about.

What you DO need to do is to ***remove of1st from this repo***, and if possible, from all commit history. It's a blatant copyright violation.

**Reply by MrZ626** [reply]
okay that make sense

[a comment by an automated account is not shown]

**Reply by C29H25N3O5** [reply]
> sounds like a mix of Bad Apple + Blank + Tetra Legends Night Of Nights

In fact just Bad Apple. It is the original version composed by ZUN in 1998 in the Touhou game *Lotus Land Story*.

**Reply by User670** [reply]
We still don't have the concrete copyright terms on Touhou songs (When and where did ZUN say his music is free to use, and under what conditions/limitations?)

**Reply by C29H25N3O5** [reply]
> We still don't have the concrete copyright terms on Touhou songs (When and where did ZUN say his music is free to use, and under what conditions/limitations?)

In fact, Zun has already specified this in a [condition published in 2020](https://thwiki.cc/东方Project使用规定案). (CN/JP)
In short, non-commercial Touhou fanwork is completely OK and we don't have to ask for any authorisation.

**Reply by User670** [reply]
As far as my understanding of the link above, as well as [this page](https://en.touhouwiki.net/wiki/Touhou_Wiki:Copyrights) goes, the use of a Touhou remix song as a video game soundtrack is not explicitly covered nor obviously implicitly covered by any of the terms.

Though I admit reading the terms more literally than an ordinary, average person would read them. I'm not good at reading between the lines.

[a comment by an automated account is not shown]
